src/components/navigation/navigation.tsx

```diff
@@ -127,7 +127,11 @@ export class Navigation {
     }
 
     private renderMenuItem(item: MenuItem) {
-        const classList = {
+        const itemClassList = {
+            active: this.isRouteActive(item.path),
+            'panel-item': true,
+        };
+        const chapterClassList = {
             active: this.isRouteActive(item.path),
```

Comment:
I mostly need your opinion about these lines Adrian. The rest is just playing around with UI and not super important at this moment; unless you dislike what you see in front-end. :)

Reply:
Sure, I'll have a look when I can make the time! 👍